README.md

```diff
@@ -1,24 +1,141 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+# Budget tracker
 
-* Ruby version
+![Grey And White Modern LinkedIn Article Cover Image](https://github.com/anbehindY/vet-clinic/assets/110251622/5dd98313-8f77-4414-9fb0-368e125e4faf)
+## 📗 Table of Contents
 
-* System dependencies
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Configuration
 
-* Database creation
 
-* Database initialization
+## Budget tracker <a name="about-project"></a>
 
-* How to run the test suite
+Budget tracker is a handy app to keep in touch with your transanctions in a much more organized way.
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+- Ruby on Rails
+- Tailwind CSS
+- Postgres
+
+### Key Features <a name="key-features"></a>
+
+- Get an organized view of categorized transactions in details
+- Add new transactions
+- Add new categories
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+- Ruby environment installed
+- Postgres database installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+cd my-folder
+git clone https://github.com/anbehindY/Budget-tracker.git
+```
+
+### Install
+
+Run `bundle install` to install all the required dependencies.
+
+### Usage
+
+To run the project:
+
+Create the database by running the following command:
+
+```sh
+rails db:create
+```
+
+Load the database schema with this command:
+
+```sh
+rails db:schema:load
+```
+
+Seed the database with the following command:
+
+```sh
+rails db:seed
+```
+
+Start the server by running the following command:
+
+```sh
+rails server
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] [REQUIRED] Good job so far! You shoud deploy the application using `Render`, `Netlify` or `GitHub Pages`.
The project should be deployed properly, the `home` and `detail` page works as expected and link to the deployed project should be present in the `README` 🙏🏼